.github/ISSUE_TEMPLATE/tutorial_request.yaml

```diff
@@ -0,0 +1,37 @@
+name: GoaT Tutorial Request
+description: Unsure about a query or feature? Request a tutorial for GoaT Wiki or Youtube Channel.
```

Comment:
Want help with a query or feature? Request a tutorial on the GoaT wiki or Youtube channel.